server/controllers/BooksController.js

```diff
@@ -0,0 +1,32 @@
+import models from '../database/models';
+import helpers from '../helpers';
+
+const { responseMessage } = helpers;
+const { Book, Author } = models;
+
+/**
+ * Add a new book
+ * @name addBooks
+ * @async
+ * @param {Object} request
+ * @param {Object} response
+ * @returns {JSON Object}
+ */
+
+class BooksController {
```

Comment:
Your controller can be named **BookController** instead of **BooksController**, same as the file name